shell/platform/android/image_external_texture_vk.h

```diff
@@ -15,6 +17,8 @@
 
 namespace flutter {
 
+static constexpr size_t kImageReaderSwapchainSize = 3u;
```

Comment:
Add comment something like:

// This value must be kept in sync with the Java variable in FlutterRenderer MAX_IMAGES.

Add also add a comment there pointing back here.